.github/workflows/go-integration.yml

```diff
@@ -29,4 +29,42 @@ jobs:
         uses: actions/upload-artifact@v2.2.4
         with:
           name: junit.xml
-          path: test_reports/junit.xml
\ No newline at end of file
+          path: test_reports/junit.xml
+
+      - name: Assemble container tag
+        id: container_tag
+        uses: pr-mpt/actions-commit-hash@v1
+        with:
+          commit: "${{ github.sha }}"
+          prefix: "${{ github.ref_name }}"
+
+      - name: Upload dev images to dockerhub
+        run: |
+          # Note(JayF): All the offical docker actions for uploading these
+          # containers also assume you need to rebuild them. That makes them
+          # ill-suited for our workflow.
+          TAG=$(echo -n "${{ steps.container_tag.outputs.long }}" | sed 's|/|-|g')
+
+          # TODO(JayF): how to get these secrets in here safely?
+          echo ${DOCKERHUB_PASS} | docker login -u ${DOCKERHUB_USER} --password-stdin
+
+          IMAGES_TO_UPLOAD=(
```

Comment:
Anyway to DRY this? Variable in GitHub actions?

Reply:
This isn't the exact set we're using in another place. It's subtly different. And frankly, I'm happy in this case to keep it all in bash.